### DISCUSSION TITLE
홈 버튼이랑 채팅 버튼을 경우에 따라 끄고 싶은데 해당 UI를 잡는 법 아시는 분 계신가요?

### DISCUSSION THREAD

**Discussion by ryusiin** [naverz/zepeto-studio-kor#800] [5. Q&A]
![image](https://user-images.githubusercontent.com/5895396/170196353-f6fc3d5a-8ced-424c-8938-37b532310393.png)

- 왼쪽 위의 홈과 채팅 버튼을 상황에 따라서 이동 시키거나 끄고 싶습니다. 
>> 해당 오브젝트를 잡는 법을 아시는 분 있나요?

- 그 외에 홈 버튼 안에 있는 기능 등을 조작할 수 있는 Docs가 어디 있는 지 아시는 분 있으면 알려주시면 감사하겠습니다.

**Comment by x4cf** [comment]
아직 제거할 수 없어요 :)
